WISE-Community/WISE-Client
# feat(Password): Allow special characters in password

## Changes

- Allow these symbols to be used in a password !@#$%^&*
- Passwords now require a letter, number, symbol, and to be at least 8 characters (we no longer require uppercase letter but they may still be used)
- Added password strength indicator
- Added visual password requirement validator

## Test

- Test with https://github.com/WISE-Community/WISE-API/pull/233

Make sure setting a password in these locations still works and validates with the new requirements
- Create teacher account
- Create student account
- Forgot teacher password
- Forgot student password
- User change password
- Teacher change student password

Closes #1049
